lib/index.js

```diff
@@ -69,25 +74,9 @@ function configure() {
   if('cert' in bedrock.config.server) {
     httpsOptions.cert = fs.readFileSync(bedrock.config.server.cert);
   }
-  let caFiles = bedrock.config.server.ca;
-  if(typeof caFiles === 'string' ||
-    (Array.isArray(caFiles) && caFiles.length > 0)) {
-    if(!Array.isArray(caFiles)) {
-      caFiles = [caFiles];
-    }
-    // all certs must be parsed individually
-    httpsOptions.ca = [];
-    caFiles.forEach(file => {
-      let bundle = fs.readFileSync(file);
-      try {
-        const certs = forge.pem.decode(bundle);
-        bundle = certs.map(cert => forge.pem.encode(cert));
-      } catch(e) {
-        throw new Error(e.message);
-      }
-      httpsOptions.ca.push.apply(httpsOptions.ca, bundle);
-    });
-  }
+  const caFiles = bedrock.config.server.ca;
+  httpsOptions.ca = _parseCerts({caFiles});
```

Comment:
So this still needs to change a bit because it's modifying previous behavior. Previously, only if `caFiles` was valid would `httpsOptions.ca` be set. Now, it will always be set, potentially to an empty array. I recommend having `_parseCerts` return undefined if the conditional in it isn't met -- and checking the return value here before setting `httpsOptions.ca`:

```suggestion
  const ca = _parseCerts({caFiles: bedrock.config.server.ca});
  if(ca) {
    httpsOptions.ca = ca;
  }
```

Reply:
Also, per the comment below, we should rename `_parseCerts` to `_readCAFiles` since we're no longer parsing.